### ISSUE TITLE
Ambiguous phrasing

### ISSUE THREAD

**Issue by Thomas-PM** [MicrosoftDocs/cpp-docs#3791]
I would argue that this sentence is ambiguous: "Linker errors occur when the same name has two different definitions in different translation units." and would rephrase it to be "Linker errors occur when the same name is defined in different translation units."

Because "two different definitions" to me implies that if a symbol is defined twice in the same exact way, it's only one definition that would be acceptable. But we all know that according to the C standard, this is not acceptable and violates the ODR. This case (one symbol redefined the same way in multiple translation units) is also one of the more common and confusing cases because that's what happens when a symbol is defined in a header file and included in multiple other files.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 04d22551-f6be-bbd0-98e6-805b8875c07e
* Version Independent ID: c94795a2-814b-0c05-d2df-5ba3bc0425c7
* Content: [Translation units and linkage (C++)](https://docs.microsoft.com/en-us/cpp/cpp/program-and-linkage-cpp?view=msvc-170)
* Content Source: [docs/cpp/program-and-linkage-cpp.md](https://github.com/Microsoft/cpp-docs/blob/main/docs/cpp/program-and-linkage-cpp.md)
* Product: **visual-cpp**
* Technology: **cpp-language**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by corob-msft** [comment]
@Thomas-PM 
Thanks for the report. An update with a fix is available now.
